src/kakarot/kakarot.cairo

```diff
@@ -31,7 +31,7 @@ func constructor{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr
 // @param bytecode The bytecode to be executed
 // @param calldata_len The calldata length
 // @param calldata The calldata which can be referenced by the bytecode
-// @return stack_accessess_len The size of the accesses array of the stack delta
+// @return stack_accesses_len The size of the accesses array of the stack delta
```

Comment:
Since you’re here, there’s `accessesses` down below

Reply:
done